src/js/utils/getDecodingInfo.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Returns decoding info for a given media configuration.
+ *
+ * @param {MediaDecodingConfiguration} configuration Media configuration.
+ * @returns {Promise<MediaCapabilitiesInfo>} Media decoding information.
+ */
+export default async function getDecodingInfo(configuration) {
+  if (navigator.mediaCapabilities) {
+    return navigator.mediaCapabilities.decodingInfo(configuration);
+  }
+
+  // If we can't use the Media Capabilities API, try using `canPlayType`.
```

Comment:
I believe we should be using [`MediaSource.isTypeSupported`](https://developer.mozilla.org/en-US/docs/Web/API/MediaSource/isTypeSupported) instead of `canPlayType` as we're using MediaSource behind the scenes IIUC.

Reply:
You're absolutely right, thank you for the correction. 👍